OpenSim/Simulation/Model/OffsetFrame.h

```diff
@@ -242,14 +248,28 @@ void OffsetFrame<C>::constructConnectors()
 }
 
 //=============================================================================
-// FRAME COMPUTATIONS
+// FRAME COMPUTATIONS must be specialized by concrete derived Offsets
 //=============================================================================
 // Implementation of Frame interface by OffsetFrame.
 template <class C>
 SimTK::Transform OffsetFrame<C>::
-calcGroundTransform(const SimTK::State& s) const
+calcTransformInGround(const SimTK::State& state) const
 {
-    return getParentFrame().getGroundTransform(s)*getOffsetTransform();
+    return SimTK::Transform(SimTK::Vec3(SimTK::NaN));
+}
+
+template <class C>
+SimTK::SpatialVec OffsetFrame<C>::
+calcVelocityInGround(const SimTK::State& state) const
+{
+    return SimTK::SpatialVec(SimTK::Vec3(SimTK::NaN));
+}
+
+template <class C>
+SimTK::SpatialVec OffsetFrame<C>::
+calcAccelerationInGround(const SimTK::State& state) const
+{
+    return SimTK::SpatialVec(SimTK::Vec3(SimTK::NaN));
```

Comment:
Good point. Intermediate implementations had me move it out early on, but now that the OffsetFrame calculation are only dependent on its parent frame kinematics, the calcualtions have been restored on `OffsetFrame` and removed from `PhysicalOffsetFrame`. Thanks!